### DISCUSSION TITLE
Skipping deployment build when certain paths or file types are changed

### DISCUSSION THREAD

**Discussion by jamesmacwhite** [craftcms/cloud#43] [Feature Requests & Enhancements]
Possibly something that can be handled by the `craft-cloud.yaml`, but the ability to prevent a full deployment occurring when certain files e.g. `.md` or non-essential commit files that don't have influence over the runtime to prevent running builds on every commit change.

Example scenario, you make a change to the README.md of a project, but this change does not need to trigger a build/release. If this was changed alongside other files say templates, or composer, then a build would run, it is only the scenario when a commit specifically has certain paths or types of file, similar to the ignore paths option in GitHub actions: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Or alternatively, using specific values in commit messages to skip deployments: https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/skipping-workflow-runs

**Comment by angrybrad** [comment]
Yeah, we use https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/skipping-workflow-runs (for Craft) all the time, and it's great.

Good feature request!
